libs/auth/oauth.go

```diff
@@ -143,6 +144,25 @@ func (a *PersistentAuth) Challenge(ctx context.Context) error {
 	return nil
 }
 
+// Remove url query and path args and fragments from the host
```

Comment:
The comment is out of date.

Reply:
Thanks, reframed the comment to talk about retaining host and scheme.